web/src/App.js

```diff
@@ -58,6 +59,9 @@ class App extends React.Component {
             <Route path="/events" render={() =>
               <Events accessToken={this.state.accessToken} />
             } />
+            <Route path="/jobs" render={() =>
+              <Jobs accessToken={this.state.accessToken} />
```

Comment:
I think we could simplify this to `<Route path="/jobs" component={Jobs} />` because we don't need to give the access token to this component

Reply:
Agreed, will fix!